# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1553015439 0
#      Tue Mar 19 17:10:39 2019 +0000
# Node ID 0e5dc2e6c838ba1b81ac7cf3eeda0d76e3554fcd
# Parent  ca5344dd5f1c3c7d705fbb5110959f51a03c77a1
* mk-doc-cache.pk: Also translate @seealso here.  (bug #55952)

diff --git a/doc/interpreter/mk-doc-cache.pl b/doc/interpreter/mk-doc-cache.pl
--- a/doc/interpreter/mk-doc-cache.pl
+++ b/doc/interpreter/mk-doc-cache.pl
@@ -66,16 +66,18 @@ sub make_texinfo_file
           ## DOCSTRINGS header ends once we find the first function.
           if ($in_header && $line =~ m/^$doc_delim/)
             {
               $in_header = 0;
             }
           next if $in_header;
           next if $line =~ /$tex_delim_pat/;
 
+          $line =~ s/\@seealso/\@xseealso/g;
+
           ## escape {}@ characters for texinfo
           $line =~ s/([{}\@])/\@$1/g
             if $line =~ m/^$doc_delim/;
 
           print {$t_file} $line;
         }
       close ($fh);
     }
